scipy/optimize/tests/test__shgo.py

```diff
@@ -1,12 +1,12 @@
 import logging
-import numpy
+import numpy as np
 import pytest
 from pytest import raises as assert_raises, warns
+from nose.tools import nottest
```

Comment:
Don't use `nose`. If you want to skip a test, use `@pytest.mark.skip(reason="This is not run because ...")`